development.html

```diff
@@ -1,4 +1,8 @@
-3.0i
+Development footer 3.2
```

Comment:
On the development footer, this is an easy way to identify whether caches have been cleared - so having it visible is slightly easier than needing to view source.